### PR TITLE
Add org.gnu.Octave

### PR DESCRIPTION
**Pull request by Alexander-Wilms**
In the other pull request I had commited the work to the master branch, not a new one based on the new-pr branch.

### CODE REVIEW

File: org.gnu.Octave.json [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+					],
+					"dest-filename": "configure"
+				}
+			],
+			"build-commands":
+			[
+				"./configure --prefix=/app && \
+				cd Qt4Qt5 && \
+				make && \
+				make install && \
```
Comment (TingPing) [inline org.gnu.Octave.json]: Can simplify this into `make -C Qt4Qt5 install`. Also don't continue lines just have separate strings.

Comment (Alexander-Wilms) [inline org.gnu.Octave.json]: Done

**Comment by TingPing** [comment]
Test build: https://flathub.org/builds/#/builders/5/builds/470

**Comment by Alexander-Wilms** [comment]
error: Commit 62f37cf8cf88e33a6c1e967df276203cc5f3ef87e9a60f3fb2c031b8e1124989: GPG signatures found, but none are in trusted keyring

**Comment by TingPing** [comment]
Test failed, KDE Sdk still isn't on the builder though it probably should be soon now that it has a version release.

> error: Commit 62f37cf8cf88e33a6c1e967df276203cc5f3ef87e9a60f3fb2c031b8e1124989: GPG signatures found, but none are in trusted keyring

Not sure where you are seeing that?

**Comment by Alexander-Wilms** [comment]
In the terminal output of the x86_64 build at step "updating dependencies".

**Comment by nedrichards** [comment]
We have the KDE SDK available now, right?

**Comment by Barthalion** [comment]
Only for i386 and x86_64 from what I understood from @alexlarsson. We will proceed with test builds and merging of Qt/KDE apps when also ARM builds are up.

**Comment by Alexander-Wilms** [comment]
There's some interest in providing Flatpaks upstream, but they'd prefer it not being hosted under the org.gnu.Octave ID here, as long as it's not actually officially supported. It was suggested to change the ID for the time being, but since Flathub is supposed to provide packages provided by upstream with their proper ID, I'll just close this pull request for now.

**Pull request closed without merging**